domains/safaa-mammed.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "safaa-mammed",
+        "email": "safaa.mammed@gmail.com"
+    },
+    "record": {
+        "CNAME": "0120762dd104775365bb3aac447cd87b.safaa-mammed.is.a.dev. cname _82de621132b5bcbb7855a38498d4e51e.mhbtsbpdnt.acm-validations.aws."
```

Comment:
this seems to be txt verification, can you like point the domain to your website?